packages/menu-bar/src/vaadin-menu-bar-buttons-mixin.js

```diff
@@ -68,22 +91,32 @@ export const ButtonsMixin = (superClass) =>
       return this.shadowRoot.querySelector('[part="container"]');
     }
 
-    /**
-     * @return {!HTMLElement}
-     * @protected
-     */
-    get _overflow() {
-      return this.shadowRoot.querySelector('[part="overflow-button"]');
+    /** @private */
+    __hasOverflowChanged(hasOverflow, overflow) {
+      if (overflow) {
+        overflow.toggleAttribute('hidden', !hasOverflow);
+      }
     }
 
     /** @private */
-    _menuItemsChanged(items) {
+    _menuItemsChanged(items, overflow) {
+      if (!overflow) {
+        return;
+      }
+
       if (items !== this._oldItems) {
         this._oldItems = items;
         this.__renderButtons(items);
       }
     }
 
+    /** @private */
+    __i18nChanged(i18n, overflow) {
+      if (overflow && i18n && i18n.moreOptions) {
+        overflow.setAttribute('aria-label', i18n.moreOptions);
```

Comment:
It looks like now it is not possible to remove `aria-label`. Passing an empty string or `null` won't do anything.

Reply:
Good catch, fixed and added a test.